### DISCUSSION TITLE
2FA

### DISCUSSION THREAD

**Discussion by Ferielkraiem2000** [community/community#110764] [GitHub Education]
### Hi everyone,

Hi everyone , i have the problem with the fact of enabling the 2FA factor in ly github account. the botton continue is not enabled

**Comment by Dilshan012** [comment]
> ### Hi everyone,
> 
> Hi everyone , i have the problem with the fact of enabling the 2FA factor in ly github account. the botton continue is not enabled

If you're having issue with allowing Two-Factor Authentication and the "Continue" button isn't enabled, try below steps,

1.Verify Your Email Address: Make sure that your e mail cope with related to your GitHub account is proven. GitHub requires a tested e mail deal with for certain actions, inclusive of enabling 2FA.

2.Clear Cache and Cookies: Sometimes, browser cache and cookies can motive problems with website capability. Try clearing your browser's cache and cookies or open a private browser page.

3.Use a Different Browser: (Sometimes, browser extensions or settings can interfere with website functionality.)

4.Update Your Browser: Ensure that you are the usage of the latest model of your net browser. Outdated browsers may not aid certain features or have compatibility issues.

5.Check for JavaScript Errors: Open your browser's developer tools (generally accessed via urgent F12) and check for any JavaScript errors at the web page. Resolving those mistakes might also restore the problem with the "Continue" button.

6.Temporarily Disable Browser Extensions: Disable any browser extensions or accessories that is probably affecting the capability of the GitHub website.

7.Try Incognito/Private Mode: Open a new incognito/personal browsing window and attempt allowing 2FA from there. This can assist rule out any browser-associated troubles.

8.Contact GitHub Support: If none of the above steps work, you can need to attain out to GitHub Support for in addition assistance. Provide them with information about the difficulty, together with any mistakes messages you are seeing.
